ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -29,6 +24,27 @@ fn deserialise_string(v: Vec<u8>) -> Result<String, DekuError> {
     }
 }
 
+/// A trait for converting in-memory data-structures into a human-readable textual format.
+///
+/// This is modelled on [`std::fmt::Display`], but a reference to the module is always passed down
+/// so that constructs that require lookups into the module's tables from stringification have
+/// access to them.
+///
+/// The way we implement this (returning a `String`) is inefficient, but it doesn't hugely matter,
+/// as the human-readable format is only provided as a debugging aid, and isn't used in
+/// "production".
+pub(crate) trait YkDisplay {
```

Comment:
I think this should be called "IRDisplay".

Reply:
Fixed in [5c20aed](https://github.com/ykjit/yk/pull/852/commits/5c20aed1293e5154a4f76c1162eae0fee68fbb1c)